### DISCUSSION TITLE
Provisioned concurrency for Now

### DISCUSSION THREAD

**Discussion by jonatansberg** [vercel/vercel#4433] [General] [answered]
Hey!
We've been trying out Now for some of our workloads, and it's been a pleasant experience overall. The ease of use, Edge Network and effortless SWR caching is superb ✨

We're having real problems with slow cold starts tho, and pre-rendering/caching everything is not a viable option for us. Using Now with Provisioned Concurrency for Lambda Functions would adress those concerns and let us have the cake and eat it too.

I've seen a few mentions about this in other threads. Are there any plans for rolling it out?

**Comment by paulogdm** [comment, marked as answer]
Hello Jonatan,

We are considering provisioned concurrency for enterprise-grade customers since this is a feature that may be relevant for a very specific and advanced audience. If you want to consider this tier, please reach out to enterprise@vercel.com.
